code-reviews/README.md

```diff
@@ -1,12 +1,52 @@
+  them to understand the changes you're proposing.
+- A great way to provide context to the reviewer is to use Github's review
+  features. For instance, if there is a line of code that you think will be
+  confusing, comment inline on code in your own PR so that the reviewer will
+  see it. You can find an [example of that in action](https://github.com/adaptdk/tufts-acsf/pull/335/files/fcfeee988c0082a24949cf18e66464fc518bc7e9#diff-8d31c81cc321d8731734e8da9e3af11a10306595cb94532b4d47c226582d9b32) on the Tufts ACSF proeject.
+- Always provide thorough testing steps so that the reviewer can test appropriately. Testing steps include:
+  - Any steps needed to get your environment into the state that should be tested
+  - Any manual setup setps that need to happen prior to testing
+  - Testing steps + expected outcomes
+
+## Work in small batches
+
+Whenever possible, favor small, frequently submitted batches of work over large infrequently submitted batches. Small batches of work are easier for a reviewer to reason about.
+
+For instance, if you are assigned a ticket that will require changes to multiple parts of the system, consider submitting several PRs, one for each part of the system. This will allow the reviwer the opportunity to review each change in isolation, allowing them to provide better feedback.
+
+Submitting small PRs early in the process also allows for early review from peers. For instance, if you aren't sure the best way to implement something, push up a WIP PR and ask for feedback from the team.
+
+Keep in mind that `main` branch should always be in a deployable state. If you submit a PR that relies on work not yet complete, ensure that merging your initial PR will not bring `main` into a non-functional state. How to acheive this is a judgement call dependent on the situation, but usually their are logical boundaries which can be used to segment what goes into a PR. If needed you can always fall back to a single, larger PR utilizing the WIP PR state on Github to allow others the opportunity to review as you go.
+
+## Reduce the number of handoffs
+
+Each handoff requires context switching and communication, all of which take time. During a review, if you see something small that needs changing, feel free to make the change while reviewing the PR. You can then push that change to the branch under reivew and comment back on the ticket calling the original author's attention to the change.
+
+When to do this is a judgment call. In general, choose the route that will help the team progress the fastest. In other words, if you can eliminate the need for a handoff by making a change yourself, this is usually the preferred option. On the other hand, if the needed change is large or intricate, it's likely better to request changes from the original dev and incur the cost of the handoff. In other words, it's a trade-off.
+
+## Democratize code review
+
+As much as possible we should share code reviews between the core team members.
```

Comment:
@jesseday Yeah I do think that the lead will end up doing the most code review overall. They have the best view of the application as a whole + they are typically playing the role of utility infielder, taking on a lot of smaller / shorter tasks while the rest of the core team takes on longer / deeper / more focused tasks.

Reply:
In an ideal world I think that review would be split evenly amongst the core team, but in reality it will likely continue to fall more heavily on the project lead. As long as we do our best to share the responsibility I think we'll be moving in the right direction.